Validation/RecoTau/python/dataTypes/ValidateTausOnRealData_cff.py

```diff
@@ -45,8 +45,23 @@
 proc.TauValNumeratorAndDenominatorRealData.visit(zttModifier)
 #-----------------------------------------
 
+#Set discriminators
+discs_to_retain = ['ByDecayModeFinding', 'CombinedIsolationDBSumPtCorr3Hits', 'IsolationMVArun2v1DBnewDMwLT', 'IsolationMVArun2v1PWnewDMwLT']
```

Comment:
Why we retain MVAIso discriminants for newDMs, but not for oldDMs which are wider used? And why we do not test ByDecayModeFindingNewDMs which looks inconsistent with tested MVAIso?

Reply:
Using 'ByDecayModeFinding', it adds all the three decay mode finding discriminators.
Regarding the other discrimininants, I have taken the one that I understood are most used. Wich one do you suggest to use? The same for the electrons/muons. 
With the strings that I have put, you will have the discriminants that are mostly used. In fact one of the request that I have received was to reduce the number of histograms and I have put just the ones that Markus/Dennis are looking at.

Reply:
OK, thanks for clarification concerning 'ByDecayModeFinding' you are right.

What concerns MVAIso discrimiants the most used ones are DB corrected old DMs, i.e. 'IsolationMVArun2v1DBoldDMwLT'. the others which are used are DB corrected with new DMs  'IsolationMVArun2v1DBnewDMwLT'. Pileup weighted (PW) discriminants are not used for standard taus. Please check with Dennis/Markus Jan/Riccardo which discrimiants should be retained.
[Side note: the latter (for new DMs) are used for boosted taus, but new trainings have not been provided for 2017 data. It will be good to understand if they are really needed and why PW MVA was better for boosted taus (PU area overlapping with other tau in a pair?)]. 

Finally, both 'MVA6' and 'MuonRejection3' should correctly map to all electron and muon rejection discriminants, respectively. However, the names will require adjustments when version of the discrimiants will be incremented and it is why I propose to use more generic patters like 'ElectronRejection' and 'MuonRejection' which should also work now and with incremented version numbers (please check). In addition 'ElectronRejection' is more human-friendly than 'MVA6'.